odoo-shippable/scripts/entrypoint_image

```diff
@@ -58,7 +58,7 @@ def docker_entrypoint():
         subprocess.call(' '.join(cmd), shell=True)
 
     # Patch to force start odoo as root
-    cmd = """sed -i "s/== 'root'/== 'force_root'/g" %s/*/openerp/cli/server.py""" % os.getenv("HOME")  # noqa
+    cmd = """sed -i "s/== 'root'/== 'force_root'/g" %s/*/*/cli/server.py""" % os.getenv("HOME")  # noqa
```

Comment:
Considering that odoo is not installed from `LINT_CHECK=1` this will fail in that environment.
Because `%s/*/*/cli/server.py` won't find it